logic/nodes.go

```diff
@@ -585,6 +593,14 @@ func createNode(node *models.Node) error {
 	if servercfg.CacheEnabled() {
```

Comment:
Add Ip to Map after node and extclient save to DB is successful

Reply:
done